### DISCUSSION TITLE
problema en la Version 22.9

### DISCUSSION THREAD

**Discussion by fralmeag** [unix4you2/practico#22] [Q&A] [answered]
Hola, me parece muy interesante tu framework y me gustaria probarlo en algo que estoy haciendo, lo instale y aparentemente todo quedo bien,  "practico framework" me conecta con la base de datos, me sale el dashboard de la app, se ve muy intuitivo y facil de manejar, me gusto lo parametriza ble que es, cree una tabla de prueba y sale bien en mi servidor mysql, después cree un formulario con los datos de esa tabla, pero no veo como visualizar el formulario creado, voy  a la opción de campos y acciones y sale la ventana en blanco, me puedes indicar como visualizo y modifico a ese formulario?

**Comment by unix4you2** [comment, marked as answer]
Hola que tal!
Así es, vas bien pero la pantalla en blanco que ves es completamente normal.
Una cosa es que hayas creado una tabla en la base de datos y otra es la forma en que esa tabla será alimentada o visualizada.

La base de datos no es más que un lugar o repositorio donde almacenarás la información y de la que luego obtendrás reportería.  Así que cuando creas un formulario asociado a una tabla en realidad él queda en modo de diseño para que precisamente en la opción que comentas tu le puedas agregar los campos vinculados a la tabla.

Un diseño de formulario para un tabla puede ser muy variable, no sólo en maquetación sino también en sus campos por lo que puede que tengas por ejemplo una tabla con 20 campos pero un formulario con sólo 10 de ellos y luego otro formulario con sólo cinco, o combinarlos, etc.

Lo que sigue en esa pantalla que tienes es que según el tipo de campo tu puedes agregar los campos al form, definir su maquetación y reglas de trabajo para que el usuario pueda alimentar tu tabla.  

Ya luego puedes agregar botones para realizar acciones comunes como guardar, actualizar, eliminar, etc.  O algunnass acciones personalizadas por ti.

Un abrazo.
